psga/sars_cov_2/ncov2019_artic.nf

```diff
@@ -176,6 +190,10 @@ process ncov2019_artic_nf_pipeline_medaka {
       mv ${file_to_update} ${file_to_update%.*}.fa
   done
 
+  for file_to_update in `ls ${output_typing}/${ncov_prefix}_${sample_id}*.*`; do
+      sed -i "s/${ncov_prefix}_${sample_id}/${sample_id}/" ${file_to_update}
+  done
```

Comment:
COMMENT NO ACTION REQUIRED:

This codebase has an over-reliance on BASH scripts that "do everything." These BASH scripts will be hard to maintain. For new projects I recommend doing this kind of thing in python as it's much easier to write maintainable Python than maintainable BASH. Obviously that is not something that can be fixed now.